### PR TITLE
`Development`: Improve code quality and consistency in FileUploadSubmissionResource

### PR DESCRIPTION
**Pull request by PatZintel**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Server: I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/server/).
- [x] Server: I added multiple integration tests (Spring) related to the features (with a high test coverage)
- [x] Server: I documented the Java code using JavaDoc style.
### Description

adapted FileUploadSubmissionResource to now adhere to new Rest Security Guidelines
### Steps for Testing

1. Log in to Artemis
2. participate with more than one account in a FileUploadExercise
3. assess your submission as a Tutor or higher

### Review Progress

- Code Review
  - [x] Review 1
  - [x] Review 2
- Manual Tests
  - [x] Test 1
  - [x] Test 2

### Test Coverage

FileUploadSubmissionResource: 92%

### CODE REVIEW

**Review by ndricimrr — DISMISSED** [review]
Tested on TS1, couldn't spot any issues

File: src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadSubmissionResource.java [hunk 1]
```diff
@@ -66,17 +68,17 @@ public FileUploadSubmissionResource(SubmissionRepository submissionRepository, R
     }
 
     /**
-     * POST /exercises/{exerciseId}/file-upload-submissions : Create a new fileUploadSubmission
+     * POST exercises/{exerciseId}/file-upload-submissions : Create a new fileUploadSubmission
```
Comment (niclasschuemann) [inline src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadSubmissionResource.java:71]:
```suggestion
     * POST exercises/:exerciseId/file-upload-submissions : Create a new fileUploadSubmission
```

Comment (niclasschuemann) [reply src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadSubmissionResource.java:71]: Even if this does not directly add a lot to the readability, I think we should work on making this uniform :)

**Review by niclasschuemann — COMMENTED** [review]
Code looks good to me, just noted down some comments 👍

**Review by argertboja — APPROVED** [review]
I checked the code and LGTM 👍🏼 Also encountered no issues while testing on TS!

**Review by imstreetspirit — APPROVED** [review]
code seems good

**Pull request merged**